src/internal.c

```diff
@@ -14845,21 +15184,80 @@ static WC_INLINE word32 UpdateHighwaterMark(word32 cur, word32 first,
 }
 #endif /* WOLFSSL_MULTICAST */
 
+/* diff must be already incremented by one */
+static WC_INLINE void DtlsUpdateWindowGTSeq(word32 diff, word32* window)
```

Comment:
This is called in two places. Is forcing inline here really appropriate?

Reply:
removed